src/main/java/controller/GameManager.java

```diff
@@ -0,0 +1,56 @@
+package controller;
+
+import model.BaseBall;
+import view.InputUserNumber;
+import view.ViewGame;
+
+public class GameManager {
+    private final int ALL_STRIKE = 3;
+    private BaseBall BaseBall;
```

Comment:
자바에서 클래스 이름, 즉 타입 이름은 대문자로 시작하죠.
객체 이름은 카멜 케이스로 작성합니다.
```suggestion
    private BaseBall baseBall;
```